test/e2e/framework/kubernetes/create-namespace.go

```diff
@@ -0,0 +1,62 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type CreateNamespace struct {
+	Namespace          string
+	KubeConfigFilePath string
+	DryRun             bool
```

Comment:
Do we have any strong use cases for dry run? In a lot of these cases so far all core step code is gated by this conditional, and when dry run is set to true, it just pulls kubeconfig

For what it's worth the prevalidate step is effectively a dry run

Reply:
cc: @huntergregory for his thoughts

Reply:
I think we can remove this for simplicity

Reply:
This was primarily used to calculate how many IP sets et cetera that npm will create

Reply:
DryRun removed